src/core/Object3D.js

```diff
@@ -170,6 +170,26 @@ Object.assign( Object3D.prototype, EventDispatcher.prototype, {
 
 	}(),
 
+	rotateOnWorldAxis: function () {
+		
+		// rotate object on axis in world space
+		// axis is assumed to be normalized
+		// method assumes no rotated parent
+
+		var q1 = new Quaternion();
+		
+		return function rotateOnWorldAxis(axis, angle) {
```

Comment:
please follow three.js white-space formatting

Reply:
Sorry, I did my best but missed the argument spacing specification. Fixed now.

Reply:
@Mugen87 may be able to answer that. In the mean time, see http://zz85.github.io/mrdoobapproves/.

Reply:
Thanks, I was feeling brave due to it being a small change but I'll make sure to do so moving forward.